app/scripts/modules/kubernetes/src/v2/kubernetes.v2.module.ts

```diff
@@ -43,7 +42,7 @@ import { KUBERNETES_V2_RUN_JOB_STAGE } from 'kubernetes/v2/pipelines/stages/runJ
 
 // React stages
 import './pipelines/stages/deployManifest/deployManifestStage';
-
```

Comment:
I think the newline was intentional here because the `manifestSelector.validator` is not a React stage